source/support.tex

```diff
@@ -1249,8 +1249,8 @@
 The header also defines numerous macros of the form:
 
 \begin{codeblock}
-  INT_[FAST LEAST]{8 16 32 64}_MIN
-  [U]INT_[FAST LEAST]{8 16 32 64}_MAX
+  INT[_FAST _LEAST]{8 16 32 64}_MIN
+  [U]INT[_FAST _LEAST]{8 16 32 64}_MAX
   INT{MAX PTR}_MIN
```

Comment:
I think just following the C standard may be better: define italic *N*
for {8 16 32 64} and just enumerate others literally. The list fits in
285 chars, while your grammer goes 299 chars. (These numbers exclude
markups, and all word separators (including indents) are counted as 1.
Expanding {8 16 32 64} goes 663 chars.)

INT*N*_MIN INT_FAST*N*_MIN INT_LEAST*N*_MIN
INT*N*_MAX INT_FAST*N*_MAX INT_LEAST*N*_MAX
UINT*N*_MAX UINT_FAST*N*_MAX UINT_LEAST*N*_MAX
INTMAX_MIN INTPTR_MIN
INTMAX_MAX INTPTR_MAX
UINTMAX_MAX UINTPTR_MAX
PTRDIFF_MIN SIG_ATOMIC_MIN WCHAR_MIN WINT_MIN
PTRDIFF_MAX SIG_ATOMIC_MAX WCHAR_MAX WINT_MAX
SIZE_MAX

From C99 7.20.1 p2
> In the following descriptions, the symbol N represents an unsigned
> decimal integer with no leading zeros (e.g., 8 or 24, but not 04 or 048).

As a bonus, some macros will become literally searchable.